deployment/commons/sso/keycloak/docker/Dockerfile

```diff
@@ -12,17 +12,20 @@
 #     Eurotech
 ###############################################################################
 
-FROM jboss/keycloak:7.0.0
+FROM registry.access.redhat.com/ubi9 AS ubi-micro-build
+RUN mkdir -p /mnt/rootfs
```

Comment:
Ths is not a published imaged but it would be better to have only `RUN` command with everything concatenated with ` && \` to create fewer layer as possible.

Each RUN instruction creates a layer with makes the final image bigger

Reply:
Thanks for the feedback, agree with you.
Updated the `Dockerfile`.